### ISSUE TITLE
Accessibility for site launch

### ISSUE THREAD

**Issue by HeatherRemacle** [bcgov/digital.gov.bc.ca#223]
Every page on our site should meet criteria for accessibility. This should include each page being tested by a person.  In addition to ensuring accessibility criteria have been met for work currently done or underway, we also need to integrate accessibility testing into our definition of done.

The accessibility toolkit for digital content should guide this work: https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit/document-creation

# Acceptance Criteria

- There is an issue ticket for every page currently developed  to complete accessibility testing, associated with this Epic
-  Any accessibility issues detected should be added as new issues, labelled bug, and placed at the top of the backlog column
- We have found a way to ensure accessibility testing is adequately done prior to pushing content live (e.g. check boxes in our issue templates, or a requirement of QA)

**Comment by mark-a-wilson** [comment]
@patricksimonian recommends using Lighthouse audit (Chrome tool).
